external.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2021, Linaro Ltd.
```

Comment:
Should be 2023

Reply:
No. The patches were first published in 2021.

Reply:
Yeah but you reworked them today so at least add 2021-2023

Reply:
sounds reasonable